### DISCUSSION TITLE
Clone forms(Tab)

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#7037] [Help]
I have a registration system where the dialog has two forms (two tabs) one for filling in personal data and another for profile, I need to create a button to add another profile form, this new form has to appear in another tab, the new one tab can be a clone of the previous one.

I tried using the code below, but it clones the entire dialog and that's not what I need. I only need the clone of the Profile forms.
  protected updateInterface() {
      super.updateInterface();
      this.cloneButton.toggle(this.isEditMode());
  }
![Perfil_Expand](https://github.com/serenity-is/Serenity/assets/7516443/732bf3ed-f176-4eaa-82c8-28b0f24c2df3)

**Comment by VictorTomaili** [comment]
@patriciozeus please do not share any contact information in discussions.

**Comment by juliermefelix** [comment]
Can anyone help?
